src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/Components/ArtistAutocomplete.tsx

```diff
@@ -156,25 +159,29 @@ export const ArtistAutoComplete: React.FC<{
     const image = option.option?.image
 
     return (
-      <Flex alignItems="center" p={1} width="100%">
-        {option.option?.image ? (
-          <Image
-            src={image?.cropped?.src}
-            srcSet={image?.cropped?.srcSet}
-            width={image?.cropped?.width}
-            height={image?.cropped?.height}
-          />
-        ) : (
-          <Box width={44} height={44} backgroundColor="black10" />
-        )}
-        <Flex flexDirection={"column"}>
-          <Text ml={1} variant="sm-display">
-            {option.text}
-          </Text>
-          <Text ml={1} lineHeight={1.5} variant="sm-display" color="black60">
-            {option.option?.formattedNationalityAndBirthday}
-          </Text>
+      <Flex justifyContent="space-between">
+        <Flex alignItems="center" p={1} width="100%">
+          {option.option?.image ? (
+            <Image
+              src={image?.cropped?.src}
+              srcSet={image?.cropped?.srcSet}
+              width={image?.cropped?.width}
+              height={image?.cropped?.height}
+            />
+          ) : (
+            <Box width={44} height={44} backgroundColor="black10" />
+          )}
+          <Flex flexDirection={"column"}>
```

Comment:
I think just adding a `pl={1}` to Flex should be enough instead of adding a margin to each Text element.
`<Flex flexDirection={"column"} pl={1}>`

Reply:
Sure, why not :) This line wasn't really updated by this PR. I've only added`{!!showChevronIcon && <ChevronRightIcon m="auto" mr={1} />}` to the component.